src/containers/ConnectPage/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import {Button} from '@fluentui/react-northstar';
+
+import {CONNECT_ACCOUNT_LINK} from '../../constants';
+
+const ConnectPage = () => {
+    return (
+        <div className='msteams-connect'>
+            <div className='msteams-connect__title'>{'Connect your account'}</div>
+            <div className='msteams-connect__description'>{'Authorize your Microsoft Teams account and broadcast your messages from Microsoft Teams to Mattermost.'}</div>
+            <a
+
```

Comment:
```suggestion
```

Reply:
this line is added due to eslint error `Expected line before comment.`